src/VisualStudio/CSharp/Impl/Snippets/SnippetFunctions/SnippetFunctionArgumentValue.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.LanguageServices.Implementation.Snippets;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.Snippets.SnippetFunctions
+{
+    internal sealed class SnippetFunctionArgumentValue : AbstractSnippetFunctionArgumentValue
```

Comment:
nit: consider naming CSharpSnippetFunctionArgumentValue  or SimpleSnippetFunctionArgumentValue.  I'm not a huge fan of the conrete type just being the abstract type's name with the 'Abstract' portion dropped.

Reply:
This is the pattern used for all other snippet functions for both C# and Visual Basic.